app/src/main/java/org/sopt/dosopttemplate/CustomDialog.kt

```diff
@@ -0,0 +1,36 @@
+package org.sopt.dosopttemplate
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import org.sopt.dosopttemplate.databinding.DialogBinding
+
+class CustomDialog: DialogFragment() {
+    private var _binding: DialogBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
```suggestion
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View{
```
반환하는 view가 null일 수 없기 때문에 ?를 지워주셔도 좋습니다!